src/Network/MQTT/Client.hs

```diff
@@ -157,7 +158,10 @@ mqttConfig = MQTTConfig{_hostname="", _port=1883, _connID="",
                         _msgCB=NoCallback,
                         _protocol=Protocol311, _connProps=mempty,
                         _connectTimeout=180000000,
-                        _tlsSettings=TLSSettingsSimple False False False,
+                        _tlsSettings=def
```

Comment:
`def` is defined as `TLSSettingsSimple False False False` so I think just using `def` here without specifying all the fields is the right thing here.